pages/api/auth/[...nextauth].ts

```diff
@@ -151,8 +150,6 @@ export const authOptions: NextAuthOptions = {
           logMessage.warn(`Could not record Signout, token corrupt : ${JSON.stringify(token)}`);
           return;
         }
-        //remove acceptable user from cache
-        await removeAcceptableUse(token?.userId as string);
```

Comment:
Is this a valid change? It was required in the PR review as something to be removed.

Reply:
Yes! it was added to circumvent the issue (acceptable use being removed at JWT initialization etc...)  raised in  unsolved questions section. This line is no longer needed.